master/buildbot/process/buildrequestdistributor.py

```diff
@@ -543,6 +547,8 @@ def createBuildChooser(self, bldr, master):
         # just instantiate the build chooser requested
         return self.BuildChooser(bldr, master)
 
-    def _quiet(self):
+    @defer.inlineCallbacks
+    def _waitForFinish(self):
         # shim for tests
```

Comment:
Was the `pass` the shim or is the function itself a shim for tests? If the former, then I think this comment should be removed now that there's content in this function.

Reply:
Fixed in a8f36ecee17f8795384df48a2048c0d7d7200f6f.